app/views/host_aggregate/add_host_select.html.haml

```diff
@@ -0,0 +1,37 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Add Host to Host Aggregate')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Host')
+      .col-md-8
+        = select_tag("host_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @host_choices.sort),
+                      "ng-model"                    => "hostAggregateModel.host_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
+          = button_tag(_("Add"),
```

Comment:
@h-kataria @himdel : on further investigation, this won't work as we were hoping based on the gitter discussion. The reusable partial hides and shows the 'add' and 'save' buttons based on whether the object exists (i.e. 'add' shows up for new records, and 'save' shows up for existing records. These forms aren't for create and update operations -- they're for adding and removing hosts. So the 'add' button won't show up where I need it, the 'save' (which these forms don't even have) will always show up, there is no 'remove' button.